### DISCUSSION TITLE
Slow map generation

### DISCUSSION THREAD

**Discussion by SenorRodriguez** [Koenkk/zigbee2mqtt#13998] [General]
When one of the nodes goes down for whatever reason, eg LQI issues (and often it gets some other nodes with it) the map generation process gets _painfully_ slow as the coordinator tries to reach them and the timeout is very long. 
How can I ignore offline devices in map generations proces altogether, or at least change the timeout to some 10 seconds or so?
I find it very difficult to debug some network issues, when checking which device is connected through which takes me 15 minutes. 
I tried disabling availability  for problematic devices - no success thou, plus i don't know whether device is online or not :(

**Comment by ggear** [comment]
+1

I find the map feature unusable for this reason, I commonly have 10+ devices that are off (eg bulbs) which means the map generation takes over 30+ min if it completes at all.
